src/ITest/java/uk/gov/companieshouse/pscstatementdataapi/config/AbstractMongoConfig.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.companieshouse.pscstatementdataapi.config;
+
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MongoDBContainer;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Mongodb configuration runs on test container.
+ */
+public class AbstractMongoConfig {
+
+    public static final MongoDBContainer mongoDBContainer = new MongoDBContainer(
+            DockerImageName.parse("mongo:4.0.10"));
```

Comment:
Should this be hard set in the code? Could we pass in as a system var maybe for when CH upgrades mongo

Reply:
Yeah I wasn't totally sure but it seems to be in the ITests for basically every project in CH so I figured it was probs easier if we set it the same way everywhere